presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/ApiScope.java

```diff
@@ -0,0 +1,28 @@
+package com.teradata.prestomanager.controller;
+
+import com.teradata.prestomanager.controller.api.ControllerConfigAPI;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public enum ApiScope
+{
+
```

Comment:
It is always a good idea to build the project before pushing the changes to github